dhall/tests/Dhall/Test/TypeInference.hs

```diff
@@ -84,25 +84,32 @@ successTest prefix = do
         Tasty.HUnit.assertEqual message resolvedExpectedType inferredType
 
 failureTest :: Text -> TestTree
-failureTest path = do
-    let skip = [ typeInferenceDirectory </> "failure/unit/MergeEmptyNeedsDirectAnnotation1.dhall"
+failureTest prefix = do
+    let skip = [ typeInferenceDirectory </> "failure/unit/MergeEmptyNeedsDirectAnnotation1"
                ]
 
-    Test.Util.testCase path skip $ do
-        let dhallPath = Test.Util.toDhallPath path
```

Comment:
I forgot to attach the `.dhall` extension here, so all failure tests would simply fail during import resolution. :(